mappings/net/minecraft/inventory/SidedInventory.mapping

```diff
@@ -1,11 +1,11 @@
 CLASS ahm net/minecraft/inventory/SidedInventory
-	METHOD a canInsertInvStack (ILbcb;Lfa;)Z
+	METHOD a canAdd (ILbcb;Lfa;)Z
```

Comment:
Any reason over canAdd and not `isValidStack` (same as normal inv), or `canAddStack`?

Reply:
It's worth noting that this method has *zero* impact on whether a stack is valid for a slot. It is specifically and only used for automation purposes, i.e. hoppers - calling it `isValidStack` would lead to significant confusion, since A. that name already exists on the superclass and serves a completely different purpose and B. it doesn't actually have anything to do with whether a stack is valid or not. It's more of a "can you access the slot you want from this side", really.

Reply:
Lack of `Stack` suffix was an oversight on my part, I'll resolve this.